README.md

```diff
@@ -255,6 +37,18 @@ Contributions are welcome and appreciated. Here's how to get started:
 
 That's it! Assuming you can complete all of these steps without any error or issues, you should be good to go.
 
+### Documentation Site
+
+The Documentation Site is a React application which is built on the Vite framework.  There are two steps to building the site:  first, the skeleton of the site is compiled and committed into this repository; second, the various artifacts are injected from a host repository into a project-specific instance of the site via [bin/featuremap docs](https://beyond-finance.github.io/feature_map/interface/docs).
+
+Compilation of the build asset is done via `npm run build` from within the [docs](./docs) folder.  This compiles the React app into a single static file which is placed in [./lib/feature_map/private/docs/index.html](./lib/feature_map/private/docs/index.html]).
+
+The documentation site may be run locally to aid in development via `bin/docs`.  It uses sample data found in [docs/src/data/sample_config.js](./docs/src/data/sample_config.js).
+
+### README Site
+
+The README site is built with Jekyll and TailwindCSS and is hosted via GitHub Pages at:  https://beyond-finance.github.io/feature_map.  It can be run locally to aid in development via `bin/readme`.
+
 #### Publication
```

Comment:
This should probably be an h3, to align with the other Development sub-sections above.